src/decide.ts

```diff
@@ -169,7 +175,13 @@ function sizeDoesMatch(resultSize, searchee) {
 	const upperBound = length + fuzzySizeThreshold * length;
 	return resultSize >= lowerBound && resultSize <= upperBound;
 }
+function isRepackOrProper(searcheeName: string, candidateName: string) {
+	console.log(
+		`${searcheeName.match(RP_REGEX)} ${candidateName.match(RP_REGEX)}`,
+	);
 
+	return searcheeName.match(RP_REGEX) === candidateName.match(RP_REGEX);
```

Comment:
The name of the function does not match this check. Function should probably be renamed to something like `releaseVersionDoesMatch`